heron/metricscachemgr/src/java/com/twitter/heron/metricscachemgr/metricscache/MetricsCacheQueryUtils.java

```diff
@@ -0,0 +1,290 @@
+// Copyright 2016 Twitter. All rights reserved.
```

Comment:
@kramasamy suggested providing internal simple interface with wrapper to adapt tmaster protobuf interface, which I agreed